api/v1alpha1/databasecluster_types.go

```diff
@@ -225,6 +228,26 @@ type Backup struct {
 	Enabled bool `json:"enabled"`
 	// Schedules is a list of backup schedules
 	Schedules []BackupSchedule `json:"schedules,omitempty"`
+	// PITR is the configuration of the point in time recovery
+	PITR PITRSpec `json:"pitr,omitempty"`
+}
+
+// PITRSpec represents a specification to configure point in time recovery for a database backup/restore.
+type PITRSpec struct {
+	// Enabled is a flag to enable PITR
+	Enabled bool `json:"enabled,omitempty"`
+	// StorageName is the name of the BackupStorage where the PITR is enabled
+	StorageName string `json:"storageName"`
```

Comment:
For consistency, maybe we should call this `BackupStorageName`, what do you think?

Reply:
Good point, updated, thx